lxd/certificates.go

```diff
@@ -16,24 +16,20 @@ import (
 	"github.com/lxc/lxd/shared"
 )
 
+func certGenerateFingerprint(cert *x509.Certificate) string {
+	return fmt.Sprintf("%x", cert.SerialNumber)
```

Comment:
sha256 of the DER (Raw) representation of a SSL certificate is what's shown by web browsers and used in the SSL trust chains, so I think it's safe to use that as our identifier.

I'm not opposed to doing the same kind of hash shortening at display time as we are with images though if your concern is screen real estate.